lib/tf_lock/tf_lock_acquire.py

```diff
@@ -62,6 +70,13 @@ def tf_lock_acquire(root_module: Path) -> ExitCode:
         # start over
 
 
+def is_pr_closed(gh_url: str) -> bool:
+    status = sh.stdout(
+        ("gh", "pr", "view", gh_url, "--json", "state", "--jq", ".state")
+    )
+    return status in ["CLOSED", "MERGED"]
```

Comment:
Use tuples for constants. They're cheaper in both time and space, but also they're actually immutable (i.e. constant).

```suggestion
def is_pr_closed(gh_url: str) -> bool:
    status = sh.stdout(
        ("gh", "pr", "view", gh_url, "--json", "state", "--jq", ".state")
    )
    return status in ("CLOSED", "MERGED")
```